src/pudl/metadata/resources/eia860.py

```diff
@@ -258,21 +258,20 @@
                 "utility_id_eia",
                 "plant_id_eia",
                 "generator_id",
-                "owner_utility_id_eia",
-                "owner_name",
-                "owner_state",
-                "owner_city",
-                "owner_country",
-                "owner_street_address",
-                "owner_zip_code",
+                "utility_name_eia",
+                "state",
+                "city",
+                "country",
+                "street_address",
+                "zip_code",
                 "fraction_owned",
                 "data_maturity",
             ],
             "primary_key": [
                 "report_date",
                 "plant_id_eia",
                 "generator_id",
-                "owner_utility_id_eia",
+                "utility_id_eia",
```

Comment:
Since `utility_id_eia` in this table refers to the **owner** rather than the **operator** we may want to give it a resource-specific metadata override at the bottom of `pudl/metadata/fields.py` so the description highlights this difference.